test/parallel/test-stdout-buffer-flush-on-exit.js

```diff
@@ -15,13 +15,11 @@ if (process.argv[2] === 'child') {
   process.exit();
 }
 
-[22, 21, 20, 19, 18, 17, 16, 16, 17, 18, 19, 20, 21, 22].forEach((exponent) => {
+[22, 16].forEach((exponent) => {
```

Comment:
@kzc If there is no bug for it to detect, yes, it will pass deterministically. However, if `stdout` is not being flushed on exit, it will not *fail* deterministically. It will fail *most* of the time. That's the (potential) problem with getting rid of all those test iterations. Of course, there is a cost/benefit calculation to make here and it could be that the right thing to do is to get rid of most or all of those iterations. (I suspect that's the case.)